pkg/manifests/config.go

```diff
@@ -156,6 +156,8 @@ type RemoteWriteSpec struct {
 	BasicAuth *monv1.BasicAuth `json:"basicAuth,omitempty"`
 	// Bearer token for remote write.
 	BearerTokenFile string `json:"bearerTokenFile,omitempty"`
+	// Authorization section for remote write
+	Authorization *monv1.Authorization `json:"authorization,omitempty"`
```

Comment:
Wondering if we should use `monv1.SafeAuthorization` here, similar to `SafeTLSConfig` below.

Reply:
:+1: